compiler/rustc_codegen_ssa/src/traits/builder.rs

```diff
@@ -595,6 +595,10 @@ pub trait BuilderMethods<'a, 'tcx>:
         funclet: Option<&Self::Funclet>,
         instance: Option<Instance<'tcx>>,
     ) -> Self::Value;
+
+    /// Mark a call instruction as a tail call (guaranteed tail call optimization)
+    /// Used for implementing the `become` expression
```

Comment:
All those references to being used to implement `become` feel a bit redundant to me.